uk.co.compucorp.civicrm.tasksassignments/js/test/mocks/fabricators/file.js

```diff
@@ -0,0 +1,30 @@
+define(function() {
+  return {
+    single: function() {
+      return {
+        '_file' : {
```

Comment:
Same as above. We should not of course overuse this, but sometimes it is reasonable to save some decent amount of lines of code.
```js
'_file' : {  'size': '10023' }
```

Please fix this in other places in this file as well.